src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -3070,490 +3115,551 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
+                                        OutputProcessor::SOVStoreType::Summed,
+                                        zoneName);
+                    SetupOutputVariable(state,
+                                        "Zone Mixing Sensible Heat Gain Energy",
+                                        OutputProcessor::Unit::J,
+                                        state.dataHeatBal->ZnAirRpt(zoneNum).MixHeatGain,
+                                        OutputProcessor::SOVTimeStepType::System,
+                                        OutputProcessor::SOVStoreType::Summed,
+                                        zoneName);
+                    SetupOutputVariable(state,
+                                        "Zone Mixing Latent Heat Loss Energy",
+                                        OutputProcessor::Unit::J,
+                                        state.dataHeatBal->ZnAirRpt(zoneNum).MixLatentLoss,
+                                        OutputProcessor::SOVTimeStepType::System,
+                                        OutputProcessor::SOVStoreType::Summed,
+                                        zoneName);
+                    SetupOutputVariable(state,
+                                        "Zone Mixing Latent Heat Gain Energy",
+                                        OutputProcessor::Unit::J,
+                                        state.dataHeatBal->ZnAirRpt(zoneNum).MixLatentGain,
+                                        OutputProcessor::SOVTimeStepType::System,
+                                        OutputProcessor::SOVStoreType::Summed,
+                                        zoneName);
+                    SetupOutputVariable(state,
+                                        "Zone Mixing Total Heat Loss Energy",
+                                        OutputProcessor::Unit::J,
+                                        state.dataHeatBal->ZnAirRpt(zoneNum).MixTotalLoss,
+                                        OutputProcessor::SOVTimeStepType::System,
+                                        OutputProcessor::SOVStoreType::Summed,
+                                        zoneName);
+                    SetupOutputVariable(state,
+                                        "Zone Mixing Total Heat Gain Energy",
+                                        OutputProcessor::Unit::J,
+                                        state.dataHeatBal->ZnAirRpt(zoneNum).MixTotalGain,
+                                        OutputProcessor::SOVTimeStepType::System,
+                                        OutputProcessor::SOVStoreType::Summed,
+                                        zoneName);
+                }
+            }
+            int fromZoneNum = state.dataHeatBal->CrossMixing(mixingRepNum).FromZone;
+            if (fromZoneNum > 0) {
+                if (RepVarSet(fromZoneNum)) {
+                    RepVarSet(fromZoneNum) = false;
+                    std::string const fromZoneName = state.dataHeatBal->Zone(fromZoneNum).Name;
```

Comment:
Same thing. Sometimes you actually do want a copy of the string, e.g., you want a copy of it in another data structure like a map, but for temporaries/shortcuts a reference is usually sufficient.